### PR TITLE
Added overview article for Aiven Console

### LINKED ISSUE #1429: Add overview article for Aiven Console
# What should we add to our content collections?

An overview of the Aiven Console, its main features and where to find everything.

# Is this something you would work on yourself?

[X] I plan to open a pull request for this

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
# What changed, and why it matters

Added an overview article for the Aiven console and included information on the new console header changes. 

Resolves #1429

### CODE REVIEW

**Comment by rsalevsky** [comment]
LGTM

File: .github/vale/dicts/aiven.dic [hunk 1]
```diff
@@ -28,6 +29,7 @@ CloudWatch
 codecs
 compliances
 Conduktor
+Console
```
Comment (ftisiot) [inline .github/vale/dicts/aiven.dic:32]: I don't think this is necessary: Aiven is already present and Console is a correct word

File: .github/vale/dicts/aiven.dic [hunk 2]
```diff
@@ -2,6 +2,7 @@
 ACL
 ACLs
 Aiven
+Aiven Console
```
Comment (ftisiot) [inline .github/vale/dicts/aiven.dic:5]: I don't think this is necessary: Aiven is already present and Console is a correct word

File: docs/tools/aiven-console.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
+Aiven Console
```
Comment (ftisiot) [inline docs/tools/aiven-console.rst:1]: I believe we need to add `Aiven Console` in the file `.githube/vale/styles/capitalization_headings.yml`

File: docs/tools/aiven-console.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- Account settings
+- Projects and services
+- Billing groups
+
+
+.. image:: /images/tools/console/console_services_switchaccount.png
+    :alt: The Services page of the Aiven Console with the account switcher menu open, showing one account and the option to create a new account. 
+
+
+Manage your user profile
```
Comment (ftisiot) [inline docs/tools/aiven-console.rst:16]: Should 
- Manage your use profile
- Manage your accounts
- Manage your projects and service
- Manage billing groups 
Be separate HowTos?

File: docs/tools/aiven-console/howto/create-accounts.rst [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+#. At the top right side of the page, click the account name and select **Create account**. 
+
+   If you don't have an account yet, click **Create account**. 
+
+#. Enter an **Account Name**.
+
+#. Select any **Projects** that you want to add to this account. You can search for projects by name.
+
+#. If you want to invite admin users to the account, select **Yes** and enter their email addresses. They will receive an email invitation with a confirmation link.
```
Comment (ftisiot) [inline docs/tools/aiven-console/howto/create-accounts.rst:17]: Is there a specific toggle where to select **Yes**?

**Review by ftisiot — DISMISSED** [review]
Hi @harshini-rangaswamy , good work!

I left some comments to address before merging.
Also I saw you added an image `images/tools/console/admin_page.png` which is not used

File: docs/tools/aiven-console.rst (hunk 3 above)
Comment (harshini-rangaswamy) [reply docs/tools/aiven-console.rst:1]: Thanks for the info, I think I trying to fix the error in the wrong file.

File: docs/tools/aiven-console.rst (hunk 4 above)
Comment (harshini-rangaswamy) [reply docs/tools/aiven-console.rst:16]: I would leave them as is for now. This page will be linked in the Console, and the high-level features in the UI are 
- User profile
- Account settings
- Projects and services
- Billing groups

I know @staceysalamon-aiven  worked on this. I will let her take this feedback in the next iteration.

**Review by rsalevsky — APPROVED** [review]

**Pull request merged**